packages/wpcom-checkout/src/checkout-version-checker.ts

```diff
@@ -0,0 +1,21 @@
+import config from '@automattic/calypso-config';
+
+export function hasCheckoutVersion( version: string ) {
+	/**
+	 * This function can be used to query the checkoutVersion parameter from the URL. Useful for A/B testing.
+	 * @param {string} version
+	 * @returns {boolean}
+	 */
```

Comment:
Can we move this docblock above the function name to be consistent with other function docs?